src/chromeServices/contentScript/site.ts

```diff
@@ -42,6 +42,7 @@ export class Site {
         const inputLang = Site.scrapeText(this.#siteConfig.input.lang);
         const outputLang = Site.scrapeText(this.#siteConfig.output.lang);
         const outputText = Site.scrapeText(this.#siteConfig.output.text);
+        var bool = Math.random() < 0.5 ? true : false;
```

Comment:
lol don't use var (they don't obey local block scoping) - use let/const instead. (In this case, `const` is fine since `bool` never changes.